src/elements/content-sidebar/__tests__/MetadataInstanceEditor.test.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { type MetadataTemplate } from '@box/metadata-editor';
+import { screen, render } from '../../../test-utils/testing-library';
+import MetadataInstanceEditor, { MetadataInstanceEditorProps } from '../MetadataInstanceEditor';
+
+describe('MetadataInstanceEditor', () => {
+    const mockCustomMetadataTemplate: MetadataTemplate = {
+        id: 'template-id',
+        fields: [],
+        scope: 'global',
+        templateKey: 'properties',
+        type: 'template-id',
+        hidden: false,
+    };
+
+    const mockMetadataTemplate: MetadataTemplate = { ...mockCustomMetadataTemplate, displayName: 'Template Name' };
+
+    const defaultProps: MetadataInstanceEditorProps = {
+        isBoxAiSuggestionsEnabled: true,
+        isUnsavedChangesModalOpen: false,
+        template: mockMetadataTemplate,
+    };
+
+    test('should render MetadataInstanceForm with correct props', () => {
+        render(<MetadataInstanceEditor {...defaultProps} />);
+
+        const templateHeader = screen.getByText(mockMetadataTemplate.displayName);
+        expect(templateHeader).toBeInTheDocument();
+    });
+
+    test('should render MetadataInstanceForm with Custom Template', () => {
+        const props = { ...defaultProps, template: mockCustomMetadataTemplate };
+        render(<MetadataInstanceEditor {...props} />);
+
+        const templateHeader = screen.getByText('Custom Metadata');
```

Comment:
where is 'Custom Metadata' in this test file? I dont see it in the any of the mocks.